### PR TITLE
Feat: 1692 tooltips

### PR DESCRIPTION
**Pull request by BCerki**
card #1692 

continued from https://github.com/bcgov/cas-cif/pull/1771/files

### CODE REVIEW

File: app/data/jsonSchemaForm/tooltipText.ts
Comment (Sepehr-Sobhani) [inline app/data/jsonSchemaForm/tooltipText.ts:1]: I love the idea of keeping all tooltip text in one place.

**Review by Sepehr-Sobhani — COMMENTED** [review]
looks great 🤩 I'm just not sure about creating snapshots from form pages; we need to change them every time we change a single line in those form pages. what do you think?!

**Comment by BCerki** [comment]
> looks great star_struck I'm just not sure about creating snapshots from form pages; we need to change them every time we change a single line in those form pages. what do you think?!

I'm not attached to the snapshot tests. I added them based on your comment in the old PR, `It would be nice to test the text of the tooltips in unit tests and make sure we won't change accidentally later. ` (jest assertions seemed like overkill for accidental text changes). Thoughts? Now that the tooltip text is separate from the ui schemas, I think it would be hard to accidentally change the text.

**Comment by Sepehr-Sobhani** [comment]
> > looks great star_struck I'm just not sure about creating snapshots from form pages; we need to change them every time we change a single line in those form pages. what do you think?!
> 
> I'm not attached to the snapshot tests. I added them based on your comment in the old PR, `It would be nice to test the text of the tooltips in unit tests and make sure we won't change accidentally later. ` (jest assertions seemed like overkill for accidental text changes). Thoughts? Now that the tooltip text is separate from the ui schemas, I think it would be hard to accidentally change the text.

My main concern was inadvertently changing certain texts, but a single unit test would ensure that they are not altered by mistake. However, the new approach of keeping them in a separate file minimizes the risk of accidental changes. We are now good to proceed.

**Review by sam-warren — APPROVED** [review]
Changes look great :smiley:

**Pull request merged**